### DISCUSSION TITLE
Question: will add-on use RPI4 RTC data ?

### DISCUSSION THREAD

**Discussion by Masterz69** [hassio-addons/app-chrony#76] [Q&A]
I have RPI4 with RTC module, providing time in case of Internet failure etc.
Question is: will Chrony add-on use RTC time to feed to LAN clients ?
Or Internet connectivity is mandatory ?

**Comment by sinclairpaul** [comment]
The addon isn't configured with the RTC options, as it is a very specific requirement.

I should also add, unless your module has GPS built in it wouldn't be a reliable time source.
